docker-compose.yml

```
@@ -1,42 +1,46 @@
-version: '2'
+version: '3.5'
 
 services:
-  ws:
-    image: navitia/chaos-ws:dev
+  chaos-ws:
+    image: ${REGISTRY_HOST}/chaos/chaos-ws:${VERSION}
+    depends_on:
+      - chaos-ws-redis
+    networks:
+      - chaos
+      - lb-common
     environment:
-      - SQLALCHEMY_DATABASE_URI=postgresql://navitia:navitia@database/chaos
-      - RABBITMQ_CONNECTION_STRING=pyamqp://guest:guest@rabbitmq:5672//?heartbeat=60
-      - NAVITIA_URL=http://api.navitia.io
-      - DOMAIN_NAME=chaos-ws.local.canaltp.fr
-      - CACHE_TYPE=redis
-      - CACHE_REDIS_HOST=redis
-      - NAVITIA_CACHE_TIMEOUT=86400
-      - NAVITIA_PUBDATE_CACHE_TIMEOUT=10
-    command: /bin/bash /var/www/Chaos/docker/honcho_launcher.sh
+      - SQLALCHEMY_DATABASE_URI=${SQLALCHEMY_DATABASE_URI}
+      - NAVITIA_URL=${NAVITIA_URL}
+      - NAVITIA_TIMEOUT=${NAVITIA_TIMEOUT}
+      - RABBITMQ_CONNECTION_STRING=${RABBITMQ_CONNECTION_STRING}
+      - CACHE_REDIS_HOST=${CACHE_REDIS_HOST}
+      - IMPACT_EXPORT_DIR=${IMPACT_EXPORT_DIR}
+      - NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE}
+    #logging:
+    # driver: "fluentd"
+    #  options:
+    #   tag: chaos.ws.http
+    #   fluentd-async-connect: "true"
```

```suggestion
    logging:
     driver: "fluentd"
      options:
       tag: chaos.ws.http
       fluentd-async-connect: "true"
```

@jmuller1984 is it good now ?
thx

---

Need to uncomment this lines